src/WebGL.ts

```diff
@@ -786,28 +786,28 @@ export declare function uniform2ui(gl: WebGLRenderingContextId, location: WebGLU
 export declare function uniform3ui(gl: WebGLRenderingContextId, location: WebGLUniformLocation, v0: GLuint, v1: GLuint, v2: GLuint): void;
 export declare function uniform4ui(gl: WebGLRenderingContextId, location: WebGLUniformLocation, v0: GLuint, v1: GLuint, v2: GLuint, v3: GLuint): void;
 
-export declare function uniform1uiv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, data: Uint32List, srcOffset: GLuint = 0,
-	srcLength: GLuint = 0): void;
-export declare function uniform2uiv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, data: Uint32List, srcOffset: GLuint = 0,
-	srcLength: GLuint = 0): void;
-export declare function uniform3uiv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, data: Uint32List, srcOffset: GLuint = 0,
-	srcLength: GLuint = 0): void;
-export declare function uniform4uiv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, data: Uint32List, srcOffset: GLuint = 0,
-	srcLength: GLuint = 0): void;
-export declare function uniformMatrix3x2fv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, transpose: GLboolean, data: Float32List,
-	srcOffset: GLuint = 0, srcLength: GLuint = 0): void;
-export declare function uniformMatrix4x2fv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, transpose: GLboolean, data: Float32List,
-	srcOffset: GLuint = 0, srcLength: GLuint = 0): void;
-
-export declare function uniformMatrix2x3fv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, transpose: GLboolean, data: Float32List,
-	srcOffset: GLuint = 0, srcLength: GLuint = 0): void;
-export declare function uniformMatrix4x3fv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, transpose: GLboolean, data: Float32List,
-	srcOffset: GLuint = 0, srcLength: GLuint = 0): void;
-
-export declare function uniformMatrix2x4fv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, transpose: GLboolean, data: Float32List,
-	srcOffset: GLuint = 0, srcLength: GLuint = 0): void;
-export declare function uniformMatrix3x4fv(gl: WebGLRenderingContextId, location: WebGLUniformLocation, transpose: GLboolean, data: Float32List,
-	srcOffset: GLuint = 0, srcLength: GLuint = 0): void;
+// @ts-ignore
```

Comment:
Silence errors about default values not valid in `declare` statements. Does AS support those in declare statements?